src/content/tutorials/connectivity/gateway-api/_index.md

```diff
@@ -0,0 +1,433 @@
+- Access to the Giant Swarm platform API for app installation
+- Basic understanding of Kubernetes networking concepts
+
+## Installation
+
+Gateway API support is provided through three apps that work together. You can install them individually or use the Gateway API Bundle for simplified deployment.
+
+### Option 1: Install Gateway API Bundle (Recommended)
+
+The easiest way to get started is using the Gateway API Bundle, which installs all required components:
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-bundle \
+  --target-namespace=org-ORGANIZATION \
+  --version=0.3.0 > gateway-api-bundle.yaml
+
+kubectl apply -f gateway-api-bundle.yaml
+```
+
+**Note**: The app target namespace is the organization namespace since it is a bundle.
+
+### Option 2: Install Components Individually
+
+If you need more control over the installation, install each component separately:
+
+#### Step 1: Install Gateway API CRDs
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-crds \
+  --target-namespace=default \
+  --version=1.5.0 > gateway-api-crds.yaml
+
+kubectl apply -f gateway-api-crds.yaml
+```
+
+#### Step 2: Install Envoy Gateway
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=envoy-gateway \
+  --target-namespace=envoy-gateway-system \
+  --version=0.3.0 > envoy-gateway.yaml
+
+kubectl apply -f envoy-gateway.yaml
+```
+
+#### Step 3: Install Gateway API Config (Optional)
+
+For a quick start with default configuration:
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-config \
+  --target-namespace=giantswarm \
+  --version=0.5.1 > gateway-api-config.yaml
+
+kubectl apply -f gateway-api-config.yaml
+```
+
+## Configuration
+
+### Using the Default Gateway
+
+When you install the Gateway API Bundle, it automatically creates a default Gateway called `giantswarm-default` that's ready to use for the entire cluster. You can verify it exists:
+
+```bash
+kubectl get gateway -n giantswarm
+```
+
+Expected output:
+
+```text
+NAME                 CLASS           ADDRESS        PROGRAMMED   AGE
+giantswarm-default   envoy-gateway   10.0.0.100     True         5m
+```
+
+This default Gateway is configured to handle traffic for your cluster's base domain (`*.CLUSTER_ID.k8s.gigantic.io`) and is ready to use immediately with HTTPRoutes.
+
+### Custom Gateway Setup (Optional)
+
+If you need additional Gateways for specific requirements, you can create custom ones:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: custom-gateway
+  namespace: giantswarm
+spec:
+  gatewayClassName: envoy-gateway
```

Comment:
```suggestion
  gatewayClassName: giantswarm-default
```

otherwise people will need to define their own EnvoyProxy CR, which bring some specific settings for our case